documentation/modules/ROOT/pages/post-processors/reselect-columns.adoc

```diff
@@ -8,58 +14,101 @@
 
 toc::[]
 
-[NOTE]
-====
-This post-processor is supported only for the SQL database connectors.
-====
-
 == Overview
+endif::community[]
+
+To improve performance and reduce storage overhead, databases can use external storage for certain columns.
+This type of storage is used for columns that store large amounts of data, such as the PostgreSQL TOAST (The Oversized-Attribute Storage Technique), Oracle Large Object (LOB), or the Oracle Exadata Extended String data types.
+To reduce I/O overhead and increase query speed, when data changes in a table row, the database retrieves only the columns that contain new values, ignoring data in externally stored columns that remains unchanged,   .
+As a result, the value of the externally stored column is not recorded in the database log, and {prodname} subsequently omits the column when it emits the event record.
+Downstream consumers that receive event records that omit required keys or values can experience processing errors.
```

Comment:
```suggestion
Downstream consumers that receive event records that omit required values can experience processing errors.
```
I don't know of any database that supports storing these types of columns as keys, so generally only non-keys are affected (unless the user is remapping non-database keys to such keys in the event structures).  It may also be worth noting that when these values are omitted by the database, Debezium actually transmits the column with a payload value that conforms to the `unavailable.value.placeholder` configured value, coerced into the appropriate field type, i.e. byte array for bytes and string for strings, key/value map for maps, etc.